documentation/disaster-recovery.md

```diff
@@ -74,6 +74,8 @@ e.g. [update namespace and deployment names as required]
 - `kubectl -n bat-staging scale deployment itt-mentor-services-staging --replicas 0`
 - `kubectl -n bat-staging scale deployment itt-mentor-services-staging-worker --replicas 0`
 
+Note: You can enable maintenance mode first, however it is still recommended to scale down the web and worker apps to prevent and side effects from occuring.
```

Comment:
Link to: Enable maintenance mode section

Typo: and side effects -> any side effects